TodoApp/Src/TodoApp.ApiServices/Wrappers/ConnectionStringWrapper.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Configuration;
+using TodoApp.Contracts.Wrappers;
+
+namespace TodoApp.ApiServices.Wrappers
+{
+    public class ConnectionStringWrapper : IConnectionStringWrapper
+    {
+        public string DefaultConnectionString { get; } = ConfigurationManager.ConnectionStrings["DefaultConnection"].ConnectionString;
```

Comment:
That's kind of ideal place for a lazy backing field, isn't it? 

While it is true that this value will be loaded during the very first request, your application might not need it beforehand and might take some time before the first request arrives. It is kind of a micro-optimization, however, I deem it relevant for our kind of task.

BTW: It would be really nice to limit the length of lines in this file, so naming delegate that would initialize the lazy backing field I mentioned above might prove quite useful.